### PR TITLE
Create new action controller generation

### PR DESCRIPTION
**Pull request by anzin**
**Description** (*)
**I've implemented**:
  1. Created the generator for the NewAction controller.
  2. Created test for the NewAction controller. 
  3. Fixed Save Controller ADMIN_RESOURCE, Fixed Save Controller Test.

image
image

**Contribution checklist** (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with integration/functional tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by VitaliyBoyko** [comment]
Please fix conflicts

**Comment by anzin** [comment]
@VitaliyBoyko I've fixed the conflicts

File: resources/fileTemplates/internal/Magento Entity New Action Controller Class.php.ft [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    /**
+     * Create new ${ENTITY_NAME} action.
+     *
+     * @return ${RESULT_INTERFACE}|${FORWARD}
+     */
+    public function execute()
+    {
+        /** @var ${FORWARD} $resultForward */
+        $resultForward = $this->resultFactory->create(${RESULT_FACTORY}::TYPE_FORWARD);
+        $resultForward->forward('edit');
```
Comment (VitaliyBoyko) [inline resources/fileTemplates/internal/Magento Entity New Action Controller Class.php.ft:32]: Did not get why do we need forward 🤔

Comment (VitaliyBoyko) [reply resources/fileTemplates/internal/Magento Entity New Action Controller Class.php.ft:32]: We expect something like this https://github.com/magento/inventory/blob/1.2-develop/InventoryAdminUi/Controller/Adminhtml/Stock/NewAction.php

**Review by VitaliyBoyko — CHANGES_REQUESTED** [review]

File: resources/fileTemplates/internal/Magento Entity New Action Controller Class.php.ft (hunk 1 above)
Comment (anzin) [reply resources/fileTemplates/internal/Magento Entity New Action Controller Class.php.ft]: @VitaliyBoyko I've wase changed generated file by your recommendation.

**Review by VitaliyBoyko — APPROVED** [review]

**Pull request merged**